bitnami/opencart/4/debian-12/rootfs/opt/bitnami/scripts/libopencart.sh

```diff
@@ -334,6 +336,8 @@ opencart_update_hostname() {
 
         opencart_conf_set HTTP_SERVER "${http_url}/administration/" "$OPENCART_ADMIN_CONF_FILE"
         opencart_conf_set HTTP_CATALOG "${http_url}/" "$OPENCART_ADMIN_CONF_FILE"
+        opencart_conf_set HTTPS_SERVER "${http_url}/administration/" "$OPENCART_ADMIN_CONF_FILE"
+        opencart_conf_set HTTPS_CATALOG "${http_url}/" "$OPENCART_ADMIN_CONF_FILE"
```

Comment:
Is setting the `HTTPS_xxx` environment variables needed for the case of HTTP ?

Reply:
I believe similar issue could happen if those are not set properly when OPENCART_ENABLE_HTTPS is *false*.
Also in this way it will be consistent with the configuration for the client area:
```
opencart_conf_set HTTP_SERVER "${http_url}/"
opencart_conf_set HTTPS_SERVER "${http_url}/"
```

Reply:
@rafariossaa were you able to check this?

Reply:
Sorry for the delay. I am checking this.